src/ui/components/shared/Dialog.tsx

```diff
@@ -15,31 +15,34 @@ export function Dialog({ children, className, ...props }: DialogPropTypes) {
       >
         {children}
       </div>
-      <div>
-        <div className="grid grid-cols-1 gap-1 pt-4 text-xs text-gray-400 place-items-center">
-          <div>
-            <div className="flex space-x-2">
-              <div>
-                <a className="hover:underline" href="http://docs.replay.io">
-                  Documentation
-                </a>
-              </div>
-              <div className="text-gray-300">•</div>
-              <div>
-                <a className="hover:underline" href="http://replay.io/discord/">
-                  Discord
-                </a>
-              </div>
-              <div className="text-gray-300">•</div>
-              <div>
-                <a className="hover:underline" href="mailto:support@replay.io">
-                  Email support
-                </a>
+
+      {showFooterLinks ? (
+        <div>
```

Comment:
I think this `div` tag and the `div` tag on line 22 are probably not doing anything, but I could be wrong.